cardano-db/test/test-db.hs

```diff
@@ -1,18 +1,37 @@
 {-# LANGUAGE OverloadedStrings #-}
 
+import           Prelude
+
 import           Test.Tasty (defaultMain, testGroup)
 
 import qualified Test.IO.Cardano.Db.Insert
 import qualified Test.IO.Cardano.Db.Migration
 import qualified Test.IO.Cardano.Db.TotalSupply
 import qualified Test.IO.Cardano.Db.Rollback
 
+import           System.Environment (lookupEnv, setEnv)
+import           System.Directory (getCurrentDirectory)
+import           System.FilePath ((</>))
+
 main :: IO ()
-main =
+main = do
+  -- If the env is not set, set it to default.
+  maybePgPassFile <- lookupEnv "PGPASSFILE"
+  _ <- whenNothing maybePgPassFile $ do
+      currentDir        <- getCurrentDirectory
+      let pgPassFile    = currentDir </> "../config/pgpass"
+      setEnv "PGPASSFILE" pgPassFile
+      return pgPassFile
```

Comment:
Why return this and then throw away the result?

Reply:
Because we have no use for it? What would we do with it?
And if somebody needs to debug this or change this, I would argue that this is the correct return result.